libs/wingsdk/src/shared-aws/function.inflight.ts

```diff
@@ -11,12 +13,49 @@ export class FunctionClient implements IFunctionClient {
   /**
    * Invoke the function, passing the given payload as an argument.
    */
-  public async invoke(payload: string): Promise<string> {
+  public async invoke(
+    payload: string,
+    logsCollector?: Trace[]
+  ): Promise<string> {
     const command = new InvokeCommand({
       FunctionName: this.functionArn,
       Payload: fromUtf8(JSON.stringify(payload)),
+      LogType: LogType.Tail,
     });
     const response = await this.lambdaClient.send(command);
+    const arnParts = this.functionArn.split(":");
+    const functionName = arnParts[arnParts.indexOf("function") + 1];
+
+    if (logsCollector) {
+      const logs: string[] =
+        atob(response.LogResult ?? "").match(
+          // date, then any char then a line of winglog
+          /(\d{4}-\d{2}-\d{2}T\d{2}:\d{2}:\d{2}.\d{3}Z)(.*)winglogstart:[\s\S]+?:winglogend/gm
+        ) ?? [];
+
+      logsCollector.push(
+        ...logs.map((item: string) => ({
+          data: {
+            message: (item.match(/(?<=winglogstart:)[\s\S]*?(?=:winglogend)/) ??
+              [])[0],
+          },
+          sourceType: FUNCTION_TYPE,
+          sourcePath: functionName, //TODO: is that the right thing to do?
```

Comment:
I think it's worth trying to be consistent with simulator (if possible) and try providing the construct path. It might be possible to do this by inserting a new parameter to the constructor in this file:

```ts
constructor(
    private readonly functionArn: string,
    private readonly constructPath: string,
    private readonly lambdaClient = new LambdaClient({})
  ) {}
```

and injecting the value when creating the inflight client:

https://github.com/winglang/wing/blob/7d9910208966d69005cc87b06186d07f23839162/libs/wingsdk/src/target-tf-aws/function.ts#L253

to
```ts
[`process.env["${this.envName()}"]`, this.node.path]
```

If it doesn't work, using the AWS function ARN sounds ok